### ISSUE TITLE
[Bug] [puppeteer-extra-plugin-recaptcha] Broken Types prevent Typescript compiling

### ISSUE THREAD

**Issue by leumasme** [berstend/puppeteer-extra#500]
**Describe the bug**

When trying to compile a Typescript project using the libaries listed below, "npx tsc" will output the following error:
`node_modules/puppeteer-extra-plugin-recaptcha/dist/index.d.ts:10:5 - error TS2610: 'opts' is defined as an accessor in class 'PuppeteerExtraPlugin', but is overridden here in 'PuppeteerExtraPluginRecaptcha' as an instance property.`
Also tested puppeteer@^5.0.0 instead of ^7.0.0
Also tested puppeteer-extra@3.1.2 [lowest 3.x.x version on npm], error persisted.
Testing with puppeteer-extra@3.1.2 [highest 2.x.x version on npm] didnt work, as it required puppeteer ^2.0.0 

**Code Snippet**

N/A - I believe that this is not related to user code, but the projects own types

**Versions**

[I can not properly run envinfo so here is a cropped output of npm list --depth=2]

+-- puppeteer-extra-plugin-recaptcha@3.4.0  
| +-- puppeteer-extra-plugin@3.1.9  
| | x-- puppeteer-extra@3.1.18 deduped  
| x-- puppeteer-extra@3.1.18 deduped  
+-- puppeteer-extra-plugin-stealth@2.7.8  
| +-- puppeteer-extra-plugin-user-preferences@2.2.12  
| | +-- puppeteer-extra-plugin-user-data-dir@2.2.12  
| | x-- puppeteer-extra-plugin@3.1.9 deduped  
| x-- puppeteer-extra-plugin@3.1.9 deduped  
+-- puppeteer-extra@3.1.18  
| +-- @types/puppeteer@5.4.3  
| x-- puppeteer@7.0.0 deduped  
+-- puppeteer@7.0.0

**Comment by TotallWAR** [comment]
Same here.
Versions:
    "puppeteer": "^5.5.0",
    "puppeteer-extra": "^3.1.18",
    "puppeteer-extra-plugin-recaptcha": "^3.4.0",


It seems to be broken here:

`export declare class PuppeteerExtraPluginRecaptcha extends PuppeteerExtraPlugin {
    opts: types.PluginOptions;
...
`

**Comment by leumasme** [comment]
Simply modifying this line to
`get opts(): types.PluginOptions;`
solves this error.

Here is a patch file (usable with npx patch-package) to fix this
`diff --git a/node_modules/puppeteer-extra-plugin-recaptcha/dist/index.d.ts b/node_modules/puppeteer-extra-plugin-recaptcha/dist/index.d.ts
index b6f8078..8352a75 100644
--- a/node_modules/puppeteer-extra-plugin-recaptcha/dist/index.d.ts
+++ b/node_modules/puppeteer-extra-plugin-recaptcha/dist/index.d.ts
@@ -7,7 +7,7 @@ export declare const BuiltinSolutionProviders: types.SolutionProvider[];
  * @noInheritDoc
  */
 export declare class PuppeteerExtraPluginRecaptcha extends PuppeteerExtraPlugin {
-    opts: types.PluginOptions;
+    get opts(): types.PluginOptions;
     private contentScriptDebug;
     constructor(opts: Partial<types.PluginOptions>);
     get name(): string;
`

**Comment by berstend** [comment]
This should now be fixed through #555:
```
Successfully published:
 - puppeteer-extra-plugin-adblocker@2.12.0
 - puppeteer-extra-plugin-anonymize-ua@2.3.0
 - puppeteer-extra-plugin-block-resources@2.3.0
 - puppeteer-extra-plugin-click-and-wait@2.3.0
 - puppeteer-extra-plugin-devtools@2.3.0
 - puppeteer-extra-plugin-flash@2.3.0
 - puppeteer-extra-plugin-font-size@2.3.0
 - puppeteer-extra-plugin-recaptcha@3.5.0
 - puppeteer-extra-plugin-repl@2.3.0
 - puppeteer-extra-plugin-stealth@2.8.0
 - puppeteer-extra-plugin-user-data-dir@2.3.0
 - puppeteer-extra-plugin-user-preferences@2.3.0
 - puppeteer-extra-plugin@3.2.0
 - puppeteer-extra@3.2.0
```
